languages/messages/wikia/MessagesZh_hant.php

```diff
@@ -6,5 +6,13 @@
 'copyrightpage' => 'w:Wikia:Licensing|Wikia:Licensing',
 'addnewtalksection-link' => '開始一個新小節',
 'tog-enableuserjs' => '啟用個人JavaScript ([[Help:Personal_CSS_and_JS#JavaScript|了解更多]])',
+
+'recentchanges-on-wikia-title' => '最近在Wikia上的變化',
+'recentchanges-on-wikia-content' => "介紹模版類型－通過類型組織您的模版，讓[[Special:Templates|模版管理]]變得更簡單，以改善它們在手機上的顯示。 [[Help:Template Types|了解更多]]或'''[[Special:Insights/templateswithouttype|即
+刻就去體驗一下！]]'''
```

Comment:
This one won't be broken since it's the piped text, but it will insert an unexpected space where the new line is.